PRIVACY_POLICY.md

```diff
@@ -0,0 +1,118 @@
+
+### What Information is Collected
+
+* GitHub Username
+* GitHub Gravatar URL
+* GitHub `node_id` (Think of your `node_id` like the random number assigned to you when making a user account. This is public information)
+
+### How is this Information Collected
+
+When you sign up with Pulsar, and choose to make an account, this is the data collected to make sure your account works, and is able to protect Packages from malicious
+Actors pretending to be someone they're not.
+
+### What is Done with this Information
+
+This information is used to let you authenticate against the Pulsar Backend, when publishing, deleted, or starring a package.
+
+### Who has Access to this Information
+
+The only people who have access to this information are Pulsar's Core Admin Team in charge of the Backend.
+
+### How Long is this Information Kept
+
+The above details will be kept until either you request deletion of your account, or there is a built in supported method to delete your Pulsar User Account.
+
+### How do I have this Information Deleted
+
+To request removal of the above details feel free to contact the Pulsar Core Admin Team, through any of the [supported methods](https://pulsar-edit.dev/community.html).
+But the recommended method would be to contact @Admins on the [Pulsar Discord](https://discord.gg/7aEbB9dGRT) or via email to admin@pulsar-edit.dev
+
+### TLDR
+
+* Creating a Pulsar User Account provides Pulsar with your GitHub name, GitHub Gravatar URL, and GitHub `node_id`.
+* This information is kept until you request deletion or there is a built in way to delete your Pulsar User Account.
+* The only people who can access this information is the Pulsar Core Admin Team.
+* This information is purely there to help you use the Pulsar Backend to publish, delete, star, or otherwise interact with Packaages.
+* Pulsar User Accounts contain **Zero** details of how you log in, or any type of API, OAuth, or other token to access your account.
+
+## Pulsar Website (`https://pulsar-edit.dev`)
+
+The Pulsar Website is a service you connect to anytime you visit our website.
+
+This Website is hosted on [GitHub Pages](https://pages.github.com/) which while conveinent for us, also means that we have no built in mechanism to access any personal data of any kind through this service.
+That is to say we don't, no analytics are setup on the website, and no extra code is run on the website to collect any data.
+
+### TLDR
+
+* The Pulsar Website Collects **Zero** Data about you, or where you are.
+* It does not Log anything, save anything, or extract anything.
+* It's just a basic website.
+
+## Pulsar Editor
+
+The main Pulsar Application, available [here](https://github.com/pulsar-edit/pulsar) is the program you use anytime you launch Pulsar, or edit text within it.
+
+While [Atom](https://atom.io) the original creator of the Editor did collect telemetry and metrics about the users, when we began work on the editor one of the first
```

Comment:
"While GitHub, the original creator of the editor, did collect telemetry..."

Reply:
This one I think is ambiguous as the repo organisation is indeed Atom even if it is GH behind that. Maybe:
> While Atom, the original version(/implementation?) of the editor, did collect...
?